xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/RendererMediaCodecSurface.cpp

```diff
@@ -143,6 +143,11 @@ bool CRendererMediaCodecSurface::RenderHook(int index)
         srcRect.x2 *= 2.0;
       break;
 
+      case RENDER_STEREO_MODE_MONO:
+        dstRect.y2 = dstRect.y2 * (dstRect.y2 / m_sourceRect.y2);
+        dstRect.x2 = dstRect.x2 * (dstRect.x2 / m_sourceRect.x2);
```

Comment:
This is: `dstRect.x2 *= dstRect.x2 / m_sourceRect.x2`.
If `dstRect` is narrow and `m_sourceRect` is wide, then `dstRect` becomes more narrow?
All instructions in this method silently assume that `.x1 = 0` and `.y1 = 0`. Is this correct?